pir/cpp/server_test.cpp

```diff
@@ -50,8 +50,11 @@ using std::vector;
 
 class PIRServerTest : public ::testing::Test {
  protected:
-  void SetUp() {
-    db_.resize(DB_SIZE);
+  void SetUp() { SetUpDB(10); }
```

Comment:
Suggestion: `(DEFAULT_POLY_MODULUS_DEGREE / 2) + 1` or something similar.

Reply:
Why? I usually prefer tests that have values that are well defined and don't change based on other factors.

Reply:
OK, the problem is about the test SingleCT which assumes that `10 <= DEFAULT_POLY_MODULUS_DEGREE`. If someone was modifying `DEFAULT_POLY_MODULUS_DEGREE=4`, this underlying assumption would not be true anymore.

So maybe the problem is not here but in the SingleCT test where you can resize the DB as in the MultiCT test.

Reply:
I've updated to set the poly_modulus_degree explicitly.